sip-servlets-impl/src/main/java/org/mobicents/servlet/sip/core/dispatchers/SubsequentRequestDispatcher.java

```diff
@@ -191,10 +191,10 @@ public void dispatchMessage(final SipProvider sipProvider, SipServletMessageImpl
 							return ;
 						} else {
 							if(poppedRouteHeader != null) {
-								throw new DispatcherException(Response.SERVER_INTERNAL_ERROR, "cannot find the application to handle this subsequent request " + request +
+								throw new DispatcherException(Response.CALL_OR_TRANSACTION_DOES_NOT_EXIST, "cannot find the application to handle this subsequent request " + request +
```

Comment:
Could you please add a comment refereing to the solved issue. Add the link to the issue, and brief explanation to why we are changing the error code. References to RFC are welcomed